.docker/detailed_sample_config/V0793__detailed_sample_data.sql

```diff
@@ -0,0 +1,14 @@
34:18'),(92,'An','Anus',1,'2017-12-06 15:31:50',1,'2017-12-06 15:31:50'),(93,'Re','Rectum',1,'2018-01-11 15:36:14',1,'2018-01-11 15:36:14'),(94,'Gi','Gastrointestinal stromal',1,'2018-01-11 15:36:15',1,'2018-01-11 15:36:15'),(95,'Us','Urine Supernatant',1,'2018-01-17 15:17:54',1,'2018-01-17 15:17:54'),(96,'Up','Urine Pellet',1,'2018-01-17 15:17:55',1,'2018-01-17 15:17:55'),(97,'Po','Peri-aorta',1,'2018-01-30 21:10:16',1,'2018-01-30 21:10:16'),(98,'Gn','Gingiva',1,'2018-01-30 21:10:17',1,'2018-01-30 21:10:17'),(99,'Tg','Thymic Gland',1,'2018-03-01 15:27:20',1,'2018-03-01 15:27:20'),(100,'Ms','Mesentary',1,'2018-03-01 15:27:20',1,'2018-03-01 15:27:20'),(101,'Ri','Rib',1,'2018-03-01 15:27:20',1,'2018-03-01 15:27:20'),(102,'Gj','Gastrojejunal',1,'2018-03-27 20:46:56',1,'2018-03-27 20:46:56'),(103,'Pc','Pancreatobiliary',1,'2018-04-16 17:03:26',1,'2018-04-16 17:03:26'),(104,'Cf','Cell-Free DNA',1,'2018-04-18 18:00:16',1,'2018-04-18 18:00:16'),(105,'Ph ','Paratracheal',1,'2018-04-23 16:57:27',1,'2018-04-23 16:57:27'),(106,'Rp','Retroperitoneum',1,'2018-04-23 16:57:27',1,'2018-08-27 15:14:07'),(107,'Ae','Adnexa',1,'2018-04-24 16:31:51',1,'2018-04-24 16:31:51'),(108,'Bw','Bowel',1,'2018-05-16 21:23:29',1,'2018-05-16 21:23:29'),(109,'At','Astrocytoma',1,'2018-05-16 21:23:29',1,'2018-05-16 21:23:29'),(110,'Ra','Right Adnexa',1,'2018-06-13 19:28:54',1,'2018-06-13 19:28:54'),(111,'Vu','Vulva',1,'2018-06-25 20:21:31',1,'2018-06-25 20:21:31'),(112,'Ue','Ureter',1,'2018-06-25 20:24:37',1,'2018-06-25 20:24:37'),(113,'Uc','Urachus',1,'2018-08-10 19:19:31',1,'2018-08-10 19:19:31'),(114,'Su','Sternum',1,'2018-09-28 16:38:54',1,'2018-09-28 16:38:54'),(115,'Rc','Rectosigmoid',1,'2018-09-28 16:53:27',1,'2018-09-28 16:53:27'),(116,'Sc','Scalp',1,'2018-09-28 16:53:27',1,'2018-09-28 16:53:27'),(117,'Ga','Gastric Margin',1,'2018-09-28 16:53:27',1,'2018-09-28 16:53:27'),(118,'Ep','Epidural mass',1,'2018-09-28 16:53:27',1,'2018-09-28 16:53:27'),(119,'Lg','Leg',1,'2018-10-18 14:51:31',1,'2018-10-18 14:51:31'),(120,'Ck','Cheek',1,'2018-10-18 14:51:31',1,'2018-10-18 14:51:31'),(121,'Ss','Sinus',1,'2018-10-18 14:51:31',1,'2018-10-18 14:51:31'),(122,'Ic','ileocecum',1,'2018-11-22 22:19:35',1,'2018-11-22 22:19:35'),(123,'Py','periampullary',1,'2018-11-22 22:19:35',1,'2018-11-22 22:19:35'),(124,'Ao','Anorectal',1,'2018-12-11 16:49:28',1,'2018-12-11 16:49:28'),(125,'Di','Diaphragm',1,'2019-01-22 17:43:44',1,'2019-01-22 17:43:44'),(126,'Il','Ileum',1,'2019-01-22 17:56:50',1,'2019-01-22 17:56:50'),(127,'Or','Orbit',1,'2019-02-04 18:27:25',1,'2019-02-04 18:27:25'),(128,'La','Lacrimal Sac',1,'2019-02-04 18:27:25',1,'2019-02-04 18:27:25'),(129,'Cj','Conjunctiva',1,'2019-02-04 18:27:25',1,'2019-02-04 18:27:25'),(130,'Tu','Tongue',1,'2019-02-04 18:27:25',1,'2019-02-04 18:27:25'),(131,'No','Nose',1,'2019-02-05 22:01:48',1,'2019-02-05 22:01:48'),(132,'Cs','Cul-de-sac',1,'2019-02-27 16:31:25',1,'2019-02-27 16:31:25'),(133,'Gt','Genital Tract',1,'2019-02-27 16:31:25',1,'2019-02-27 16:31:25');
```

Comment:
All the dates should probably be `CURRENT_TIMESTAMP`, but probably not worth fixing.